sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/InMemoryFunctionCatalog.scala

```diff
@@ -25,7 +25,7 @@ import scala.collection.JavaConverters._
 import org.apache.spark.sql.catalyst.analysis.{NoSuchFunctionException, NoSuchNamespaceException}
 import org.apache.spark.sql.connector.catalog.functions.UnboundFunction
 
-class InMemoryCatalog extends InMemoryTableCatalog with FunctionCatalog {
+class InMemoryFunctionCatalog extends InMemoryTableCatalog with FunctionCatalog {
```

Comment:
It looks weird. function catalog extends both table catalog and function catalog?

Reply:
Oh, I see, sorry I misunderstood the code previously, closing.